apps/test/unit/templates/textResponses/TextResponsesTableTest.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import {mount} from 'enzyme';
+import {expect} from '../../../util/configuredChai';
+import TextResponsesTable from '@cdo/apps/templates/textResponses/TextResponsesTable';
+
+const responses = [
+  {
+    puzzle: 2,
+    question: "Free Response",
+    response: "Lorem ipsum dolor sit amet, postea pericula",
+    stage: "Lesson 1",
+    studentId: 1,
+    studentName: "Student A",
+    url: "http://fake.url"
+  },
+  {
+    puzzle: 3,
+    question: "Free Response",
+    response: "Lorem ipsum dolor sit amet, postea pericula",
+    stage: "Lesson 1",
+    studentId: 3,
+    studentName: "Student C",
+    url: "http://fake.url"
+  },
+  {
+    puzzle: 1,
+    question: "Free Response",
+    response: "Lorem ipsum dolor sit amet, postea pericula",
+    stage: "Lesson 1",
+    studentId: 2,
+    studentName: "Student B",
+    url: "http://fake.url"
+  },
+];
+const sectionId = 1;
+
+describe('TextResponsesTable', () => {
+  it('renders a table', () => {
+    const wrapper = mount(
+      <TextResponsesTable
+        responses={responses}
+        sectionId={sectionId}
+      />
+    );
+
+    expect(wrapper.find('table')).to.exist;
+  });
+
+  it('renders responses as table rows', () => {
+    const wrapper = mount(
+      <TextResponsesTable
+        responses={responses}
+        sectionId={sectionId}
+      />
+    );
+
+    const responseRows = wrapper.find('tbody').find('tr');
+    expect(responseRows).to.have.length(3);
+  });
+
+  it('sorts responses by student name upon clicking student name header cell', () => {
```

Comment:
I probably wouldn't bother testing this since sorting is tested within reacttabular, so on our side, sorting is just a setting, but it's probably fine to leave it in.